docs/src/languages/php/tuning.md

```diff
@@ -30,24 +30,24 @@ The most common issue is not allowing the right cookies as part of the router ca
 Some cookies, such as session cookies, need to be allowed,
 whereas others, such as marketing and analytics cookies,
 usually shouldn't be allowed to be part of the cache key.
-See the [router cache](/configuration/routes/cache.md) documentation
-and the [cookie entry](/configuration/routes/cache.md#cookies).
+See the [router cache](../../configuration/routes/cache.md) documentation
```

Comment:
Personally, I'd add a bit more context:
` For more information, see the [router cache](../../configuration/routes/cache.md) documentation ...`

Reply:
This wasn't a change to the text, just the link itself. But I made a smaller addition.